### PR TITLE
fix in saveMaps & reconstructMapFromJson

### PR DESCRIPTION
**Pull request by 7malikk**
While going through the latest code, I noticed some minor bugs in `saveMaps.js` and `reconstructMapFromJson()`, nothing major

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in a uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

### CODE REVIEW

**Comment by 7malikk** [comment]
@jywarren Could you review this?

File: examples/js/savedMaps.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ saveMapModalBtn.addEventListener('click', () => {
   const savedMaps = JSON.parse(localStorage.getItem('savedMaps'));
   bootstrap.Modal.getInstance(savedMapsModal).show();
 
-  if (savedMaps.length > 0) {
+  if (savedMaps !== null) {
```
Comment (7malikk) [inline examples/js/savedMaps.js:10]: @jywarren I noticed that when there are no `savedMaps`, the previous check return an error: 
`TypeError: Cannot read properties of null (reading 'length')`
As the `savedMaps` variable would be `null` if there were no maps in the localStorage
And the error prevented the "No Saved Maps" text from displaying (that's how I caught it)
So with this fix, it works fine

File: examples/js/savedMaps.js [hunk 2]
```diff
@@ -45,6 +45,6 @@ saveMapModalBtn.addEventListener('click', () => {
       mapList.append(eachMap);
     });
   } else {
-    mapList.innerHTML += '<h5> No Saved Maps </h5>';
+    mapList.innerHTML = '<h5> No Saved Maps </h5>';
```
Comment (7malikk) [inline examples/js/savedMaps.js:48]: This was an oversight on my part the `+` should not have been there

File: examples/js/archive.js [hunk 3]
```diff
@@ -281,87 +280,121 @@ function isJsonDetected(url) {
...
+      corners: reOrderedCorners,
+    });
   }
 
   map.imgGroup.addLayer(image);
 }
 
 // Reconstruct Map from JSON URL (image objects without coordinates are ignored)
-const reconstructMapFromJson = async (jsonDownloadURL) => {
+const reconstructMapFromJson = async (jsonDownloadURL = false, savedMapObj) => {
```
Comment (7malikk) [inline examples/js/archive.js:312]: @jywarren Here, all I did was make `reconstructMapFromJson()`  reusable, for some reason by making `jsonDownloadURL` a default parameter

I wonder why it shows other code i didn't add 🤔

Comment (jywarren) [reply examples/js/archive.js:312]: it looks like it's showing formatting changes from an auto-formatter. do you have one set up?

File: examples/js/archive.js [hunk 4]
```diff
@@ -281,87 +280,121 @@ function isJsonDetected(url) {
...
+      const options = {
+        tooltipText: imageObj.tooltipText,
+        corners: imageObj.nodes,
+      };
+      placeImage(imageURL, options, false);
+    }
   }
-}
+
+  // creates multiple images - this applies where multiple images are to be reconstructed
```
Comment (jywarren) [inline examples/js/archive.js:366]: So this until line 398 - is it part of your fixes, or is this code you're not sure where it came from?

File: examples/js/archive.js (hunk 3 above)
Comment (7malikk) [reply examples/js/archive.js:312]: Oh that's true, I must've forgotten to turn it off
I'll see how to revert it

File: examples/js/archive.js (hunk 4 above)
Comment (7malikk) [reply examples/js/archive.js:366]: Yea, this until 398 is part of my fixes

**Comment by jywarren** [comment]
Hi Malik, so is this ready to merge or are there still sections you're unsure of? It's ok to leave the formatting this time!

**Comment by 7malikk** [comment]
Oh, if we're leaving the formatting, then I believe everything is fine 
Thanks @jywarren

**Pull request merged**